Bejebeje.Services/Services/Interfaces/IImagesService.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Bejebeje.Services.Services.Interfaces
+{
+  using System.Threading.Tasks;
+
+  public interface IImagesService
+  {
+    Task<byte[]> GetArtistImageBytesAsync(string artistSlug);
```

Comment:
as in Bejebeje.Services/Services/ImagesService.cs.
`Task<bool> GetArtistImageBytesAsync(string artistSlug, out byte[] data);` (or was async + out forbidden?)